### ISSUE TITLE
Error build with Cocoapods

### ISSUE THREAD

**Issue by CesarDenis** [luggit/react-native-config#340]
I have Cocoapods installed in my project, I made the installation according to PR #329 of @muhozi and from the [comment](https://github.com/luggit/react-native-config/issues/125) of @clarsen in the issue #125 and when trying to execute the project the following error:

```
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening CheilTracker.xcworkspace

2019-03-20 02:13:06.455 xcodebuild[80368:7754213]  DTDeviceKit: deviceType from 422ed1da6cb0fdca73a040b830d57bca41096979 was NULL
2019-03-20 02:13:06.653 xcodebuild[80368:7754213]  DTDeviceKit: deviceType from 422ed1da6cb0fdca73a040b830d57bca41096979 was NULL
** BUILD FAILED **


The following commands produced analyzer issues:
        Analyze /Volumes/Files/repositores/smy/cheil/cheil-tracker/CheilTracker/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp normal x86_64
        Analyze /Volumes/Files/repositores/smy/cheil/cheil-tracker/CheilTracker/node_modules/react-native/React/Base/RCTModuleMethod.mm normal x86_64
(2 commands with analyzer issues)

The following build commands failed:
        CompileC /Volumes/Files/repositores/smy/cheil/cheil-tracker/CheilTracker/ios/build/CheilTracker/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/React.build/Objects-normal/x86_64/RCTJavaScriptLoader.o /Volumes/Files/repositores/smy/cheil/cheil-tracker/CheilTracker/node_modules/react-native/React/Base/RCTJavaScriptLoader.mm normal x86_64 objective-c++ com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

My versions are:
```
"react-native": "0.59.1",
"react-native-config": "^0.11.7",
```

My Podfile:
```ruby
platform :ios, '11.0'


target 'CheilTracker' do
projects
  use_frameworks!

  pod 'React', :path => '../node_modules/react-native'
  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  # Add new pods below this line
  pod 'react-native-config', :path => '../node_modules/react-native-config'


  target 'CheilTrackerTests' do
    inherit! :search_paths
    # Pods for testing
    # Add new pods below this line
    pod 'react-native-config', :path => '../node_modules/react-native-config'

  end
end

target 'CheilTracker-tvOS' do
projects
  use_frameworks!
  target 'CheilTracker-tvOSTests' do
    inherit! :search_paths
    # Pods for testing
  end
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == 'react-native-config'
      phase = target.project.new(Xcodeproj::Project::Object::PBXShellScriptBuildPhase)
      phase.shell_script = "cd ../../"\
                           " && RNC_ROOT=./node_modules/react-native-config/"\
                           " && export SYMROOT=$RNC_ROOT/ios/ReactNativeConfig"\
                           " && export BUILD_DIR=$RNC_ROOT/ios/ReactNativeConfig"\
                           " && ruby $RNC_ROOT/ios/ReactNativeConfig/BuildDotenvConfig.ruby"

      target.build_phases << phase
      target.build_phases.move(phase,0)
    end
  end
end
```

Can someone help me?

Graciously

**Comment by thanhnd1o2** [comment]
I have the same issue. Can any one help?

**Comment by fabriciosautner** [comment]
I have the same issue :(

**Comment by Abhi-Tech9** [comment]
@thanhnd1o2 @fabriciosautner did you guys found the solution ?

**Comment by CesarDenis** [comment]
@Abhi-Tech9, not yet and you?

**Comment by Abhi-Tech9** [comment]
> @Abhi-Tech9, not yet and you?

I started using react-native-dotenv

**Comment by fabriciosautner** [comment]
All projects I'm avoiding using Cocoapods because of the number of problems

**Comment by maxkomarychev** [comment]
hey @CesarDenis , please try my pr: https://github.com/luggit/react-native-config/pull/349

it's not required anymore to add custom lines to podfile

**Comment by pedro** [comment]
:+1: #349 has been merged/released with `0.12`
